src/bank.rs

```diff
@@ -202,6 +203,19 @@ impl Bank {
         {
             let option = bals.get_mut(&tx.from);
             if option.is_none() {
+                if let Instruction::NewVote(_) = &tx.instruction {
```

Comment:
Switch on the string to make it a two-liner?

Reply:
@garious the counters are different statics, they hold on and aggregate state so we don't ping influx on every request. so we need one per event kind.